lib/developer_portal/app/views/developer_portal/admin/applications/keys/_key.html.erb

```diff
@@ -1,6 +1,10 @@
 <%= content_tag(:tr, :id => "application_key_#{key.to_param}", 'data-key' => key.to_param) do %>
-  <td class="key"><%= key %></td>
+  <td class="key">
```

Comment:
This is to match the style of the already existing elements.
Compare before:
![Screenshot from 2025-03-05 14-54-10](https://github.com/user-attachments/assets/ccb397ee-393d-41cc-940c-9a453ce72225)

and after:
![Screenshot from 2025-03-05 15-25-31](https://github.com/user-attachments/assets/1ed10a54-0248-45a3-8840-358559473f93)